src/main/java/frc4388/robot/subsystems/Location.java

```diff
@@ -0,0 +1,40 @@
+package frc4388.robot.subsystems;
+
+import frc4388.robot.subsystems.Apriltags;
+
+public class Location {
+    final Apriltags Apriltag = new Apriltags();
+
+    public boolean isLimelight = false;
+    public boolean isApriltag = false;
+
+    //Determines which source to get pos and rot from and also resets
+    public void reoderPrio(){
```

Comment:
This is a really good function, but by making it public we put the responsibility of deciding when to reprioritize onto someone else, somewhere else. A function like this should be entirely encapsulated within the Location module. I have a bit of a rant on API design that is very opinionated if anyone wants to here it later. Anyway, I suggest calling when getting location and rotation.